bgpd/bgp_rpki.c

```diff
@@ -1656,6 +1713,46 @@ DEFPY (show_rpki_cache_connection,
 	return CMD_SUCCESS;
 }
 
+DEFPY(show_rpki_configuration, show_rpki_configuration_cmd,
+      "show rpki configuration [json$uj]",
+      SHOW_STR RPKI_OUTPUT_STRING
+      "Show RPKI configuration\n"
+      JSON_STR)
+{
+	struct json_object *json = NULL;
+
+	if (uj) {
+		json = json_object_new_object();
+		json_object_boolean_add(json, "enabled",
+					!!listcount(cache_list));
+		json_object_int_add(json, "serversCount", listcount(cache_list));
+		json_object_int_add(json, "pollingPeriodSeconds",
+				    polling_period);
+		json_object_int_add(json, "retryIntervalSeconds",
+				    retry_interval);
+		json_object_int_add(json, "expireIntervalSeconds",
+				    expire_interval);
+
+		vty_json(vty, json);
+
+		return CMD_SUCCESS;
+	}
+
+	vty_out(vty, "rpki is %s",
+		listcount(cache_list) ? "Enabled" : "Disabled");
+
+	if (list_isempty(cache_list))
```

Comment:
If the list is empty (= disabled) we need to print a new line before returning.

Reply:
fixed